### ISSUE TITLE
Finish Time not being shown

### ISSUE THREAD

**Issue by EricDimitri** [bambulab/BambuStudio#5028]
### Bambu Studio Version

Beta 1.10.0.74

### Where is the application from?

Bambu Lab github releases

### OS version

Windows 11

### Printer

Bambu Lab P1S AMS

### How to reproduce

Print something, then check the Device Tab. "Finish Time:" label is there, but no time being shown after that

### Actual results

Finish time not being Shown

### Expected results

Finish time should be shown after the label

### Project file & Debug log uploads

Project file
[PROJECT.zip](https://github.com/user-attachments/files/17424869/PROJECT.zip)
![ScreenShot](https://github.com/user-attachments/assets/575836e3-d595-41e8-827a-c37986810702)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by MartinChan3** [comment]
[1.10 beta2](https://github.com/bambulab/BambuStudio/releases/tag/v01.10.00.81) has solved this problem.

**Comment by EricDimitri** [comment]
Thanks a lot!!!

**Comment by EricDimitri** [comment]
I just installed the latest beta version, and still cant see  the Finish time. Please tell me what do you need from my side. Thanks!!!

**Comment by DanBao-Bambu** [comment]
We apologize for any inconvenience caused and have added logs to the code to track this issue. But the log level needs to be set to trace. Here are the methods:
Enter the Configuration folder
![image](https://github.com/user-attachments/assets/9114d807-d868-437a-babc-b8dab8e705a1)
Use a text editor to edit BambuStudio.conf, locate the 'severity level' field, and change it to 'trace'
![image](https://github.com/user-attachments/assets/1f9e4499-3a02-42d5-9aa0-245b870c49a5)
Then open BambuStudio and when the finish_time does not display, copy the log and send it to me as an attachment. Thank you very much.

**Comment by EricDimitri** [comment]
Ok @DanBao-Bambu , will do, but keep in mind that it NEVER shows the finish_time to me. Anyway, I will send you logs in trace lvl ASAP. Thanks a lot!!

**Comment by EricDimitri** [comment]
[debug_Mon_Nov_04_08_44_01_20740.log.zip](https://github.com/user-attachments/files/17617688/debug_Mon_Nov_04_08_44_01_20740.log.zip)
Is this the log you need? Let me know please. Thanks!!!

**Comment by EricDimitri** [comment]
Weird thing, right after I modified the log lvl in the CFG file, sent to print, and appeared... I dont know if it is related or what, but.... Really weird....

**Comment by EricDimitri** [comment]
*** Update ***
It works just sometimes... sometimes ut shows, sometimes it doesnt...  I dont know related to what.

**Comment by MartinChan3** [comment]
Can you offer a video or gif to show in which situation it doesn't work ? We should know more details...

**Comment by EricDimitri** [comment]
As far as I tested, some prints show it, some prints dont show it. How a video is helpful? I can record one but, You will see it not being shown, as in the Screenshot :)

**Comment by MartinChan3** [comment]
@EricDimitri I had print  with my A1 and P1P by Studio V1.10 Beta2, and I didn't figure out in which situation this could happen...So I asked the video so I can test like you.

**Comment by EricDimitri** [comment]
@MartinChan3 I am doing nothing weird, just send to print from Studio... Sometimes it shows, sometimers it doesnt. Couldnt figure oput the pattern for it. Next time It doesnt show it, will share the project for you to test.

**Comment by DanBao-Bambu** [comment]
![image](https://github.com/user-attachments/assets/56fb9716-db47-4d55-a597-213668f49f5e)
> Is this the log you need? Let me know please. Thanks!!!

@EricDimitri  I have checked your log and did not find any abnormalities, which is strange. Perhaps this log records when you were displaying the finish time normally?

**Comment by EricDimitri** [comment]
I saw the same... But no. I deleted log, then send to print, and then after checking it is not being shown on the screen, saved the log... Weird...

**Comment by EricDimitri** [comment]
![Weird Finish Time](https://github.com/user-attachments/assets/57ae36ed-47e6-4c90-9977-5215a92fafcf)
Look at this. Only thing I might think about is that it will finish the next day? But something is not ok there :)

**Comment by HomerSimpy** [comment]
1.10.0.89 shows line through on higher Rez monitors
![Screenshot 2024-11-12 at 6 36 21 PM](https://github.com/user-attachments/assets/fe3707df-1aa6-47cd-94df-86e0ee08f17d)

**Comment by RosomakPL** [comment]
Same here but version 1.10.1.50
![Finisch time - Zrzut ekranu 2024-11-28 140403](https://github.com/user-attachments/assets/f4571329-d163-41cb-965b-14e14313d46d)

**Comment by Rich-b1** [comment]
I'm having this same problem with Studio 1.10.1.50
Finish Time under the Progress bar will not display a time of day. 
But it will change to "Finished" when the print is done.

I have TWO computers, but only ONE X1C printer.

1. On my PC which is running Windows 10, Finish time will NOT display.
2. On my LAPTOP running Windows 11, with the same version of Studio. finish time displays OK!


I tried several things such as:

1. Connect both computers via Ethernet (previously the laptop was via WiFi but the PC was via Ethernet).
2. Uninstalled Studio, rebooted PC, Re-installed Studio.
3. Logged out of my Bambu account, and then logged in again.

Nothing seems to allow the PC to display the Finish Time.

Please advise if I should submit some logs or if this is a known issue being actively worked on already? I don't want to duplicate your efforts.

**Comment by Rich-b1** [comment]
Please see my edited post. I found that by changing my Windows Screen Scale from 100% to another value (such at 125%) works around the issue and allows the Finish time to be displayed.

**Comment by EricDimitri** [comment]
> Please see my edited post. I found that by changing my Windows Screen Scale from 100% to another value (such as 125%) works around the issue and allows the Finish time to be displayed.

Thats cool but, Just to see that number, change the scale of the whole monitor, is not worth it. It has to be fixed "right". anyway, thanks for the workaround!! And Happy New Year!!!

**Comment by JohnSmithCYM** [comment]
@Rich-b1 Thanks for your test 🐱 and we will check if it works on 1080P 100% in our internal version.

**Comment by Rich-b1** [comment]
@DanBao-Bambu @JohnSmithCYM I just purchased a new monitor. This can have a resolution of **5120 x 1440**. Even with this large resolution, if the screen "Scale and layout" is set to **100%**, Finish time is **NOT** displayed!

If I choose other Scale settings in Windows (125%, 150%, 175%, 200%, 225%), then Finish time **WILL** display ok!

So it seems it's not that the screen layout doesn't have enough room for the Finish time to be displayed, but something about the Scale and layout being at 100% that causes the problem.

Note that I'm currently using Windows 10 and have I'm using a nvidea GFORCE GTX980Ti GPU card.

I hope this helps you! If you want me to do any more testing, please let me know!

**Comment by Bladevane** [comment]
I totally agree with the above comments - the user should not have to change the screen display to see the remaining time. Us users need a fix from Bambu Labs, not a Windows workaround.

**Comment by Rich-b1** [comment]
> I totally agree with the above comments - the user should not have to change the screen display to see the remaining time. Us users need a fix from Bambu Labs, not a Windows workaround.

Personally, I don't think anyone is suggesting using a work around. But the work around lets you see the Finish time until they release a fixed version of Studio.

It sure would be nice if someone from BL would acknowledge that they have duplicated this issue, and the fix will be in the next release. But I'm not sure if the previous comments show that they understand the root cause yet.

**Comment by Bladevane** [comment]
Thanks for the reply. Yes, it would be appreciated if BL could at least acknowledge there is an issue which needs to be addressed. Maybe they are looking into it, who knows??

**Comment by Rich-b1** [comment]
I also found this is occurring in Bambu Studio Beta Version 1.10.2.64 and on Windows 11 (previous for me was Windows 10).

If the Windows Screen Scale is set to 100%, then Finish Time is NOT displayed:

![Image](https://github.com/user-attachments/assets/4c0bcdbf-5c3f-4ebe-8fac-b228e78c0288)

If the Windows Screen Scale is set to 125$, then Finish Time is displayed OK:

![Image](https://github.com/user-attachments/assets/8b9da7e9-6d5a-47e5-ac6f-a2508b492631)

**Comment by LiamDroog** [comment]
Seconding this occurring on 2.0.0.95. Windows 10,  2560x1440 monitor. One might think that BBL could fix such an easy bug in six months...

![Image](https://github.com/user-attachments/assets/c4f14d35-7a72-4219-9dfd-2971ca9c2bd8)

**Comment by EricDimitri** [comment]
@DanBao-Bambu  Loos like this was fixed on the beta, and they didnt mention this bug, that was reported way before hahahahahahaha

**Comment by Rich-b1** [comment]
the finish time is finally being shown in version 2.2.0.85! Thanks for finally fixing this issue.

**Comment by EricDimitri** [comment]
Hahahahahaah you are right @DanBao-Bambu you are probably right.

Anyway, it is not about recognition, it is about fixing bugs. Great!!!

**Comment by supertechnoboy** [comment]
Yes! Fixed for me as well in 2.2.0.85.
